pandapower/create.py

```diff
@@ -4574,47 +4489,61 @@ def create_shunt(net, bus, q_mvar, p_mw=0., vn_kv=None, step=1, max_step=1, name
     if vn_kv is None:
         vn_kv = net.bus.vn_kv.at[bus]
 
-    entries = dict(zip(["bus", "name", "p_mw", "q_mvar", "vn_kv", "step", "max_step", "in_service"],
-                       [bus, name, p_mw, q_mvar, vn_kv, step, max_step, in_service]))
+    entries = dict(zip(["bus", "name", "p_mw", "q_mvar", "vn_kv", "step", "max_step", "in_service",
+                        "step_dependency_table", "id_characteristic_table"],
+                       [bus, name, p_mw, q_mvar, vn_kv, step, max_step, in_service, step_dependency_table,
+                        id_characteristic_table]))
     _set_entries(net, "shunt", index, **entries, **kwargs)
 
+    _set_value_if_not_nan(net, index, id_characteristic_table, "id_characteristic_table",
+                          "shunt", dtype="Int64")
+
     return index
 
 
-def create_shunts(net, buses, q_mvar, p_mw=0., vn_kv=None, step=1, max_step=1, name=None,
-                  in_service=True, index=None, **kwargs):
```

Comment:
Not to be adressed in this pull request, but boa this file needs to be split up.